presentation/voca/src/main/java/com/hilingual/presentation/voca/component/AddVocaButton.kt

```diff
@@ -0,0 +1,52 @@
+package com.hilingual.presentation.voca.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.theme.HilingualTheme
+
+@Composable
+internal fun AddVocaButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier
+            .background(
+                shape = RoundedCornerShape(8.dp),
+                color = HilingualTheme.colors.black
+            )
+            .width(IntrinsicSize.Max)
+            .padding(12.dp)
+            .noRippleClickable(onClick = onClick),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = "일기 쓰고 단어 추가하기",
+            style = HilingualTheme.typography.bodySB16,
+            color = HilingualTheme.colors.white
+        )
+    }
```

Comment:
```suggestion
Text(
        text = "일기 쓰고 단어 추가하기",
        style = HilingualTheme.typography.bodySB16,
        color = HilingualTheme.colors.white,
        modifier = modifier
            .clip(RoundedCornerShape(8.dp))
            .background(
                shape = RoundedCornerShape(8.dp),
                color = HilingualTheme.colors.black
            )
            .width(IntrinsicSize.Max)
            .padding(12.dp)
            .noRippleClickable(onClick = onClick),
        textAlign = TextAlign.Center
    )
```

이렇게도 가능합니다ㅎㅎ 컴포넌트 트리를 줄일수잇어요

Reply:
오호 Column을 안쓰고도 사용할 수 잇군요 !!!!! 넘 좋네요 수정해두겟습니다 !!